src/App/App.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+import './App.scss';
+import { BrowserRouter as Router, Route, Switch, Redirect } from 'react-router-dom';
+
+import { BasicLayout, FixedToolbarLayout, GuideLayout } from './Layouts';
+
+import Home from '../containers/Home';
+import Roadmap from '../containers/Roadmap';
+import Contribute from '../containers/Contribute';
+import Guide from '../containers/Guide';
+import ConfirmationValidatorAPI from '../containers/ConfirmationValidatorAPI';
+import BankAPI from '../containers/BankAPI';
+import PrimaryValidatorAPI from '../containers/PrimaryValidatorAPI';
+import DeploymentGuides from '../containers/DeploymentGuides';
+
+/**
+ * This component wraps the react-router Route component. This is so
+ * we can specify a custom layout for each route.
+ * 
+ * Usage: <UsePage path="" component={YourPage} layout={YourLayout} />
+ */
+const UsePage = ({ component: Component, layout: Layout, ...rest }) => {
+  return (
+    <Route {...rest} render={(props) =>
+      <Layout {...props}>
+        <Component {...props} />
+      </Layout>
+    }/>
+  );
+}
```

Comment:
This is not too friendly with Typescript, I will try to come up with an alternative